server/actions_apis.go

```diff
@@ -73,7 +73,7 @@ func (s *RegistryServer) CreateApi(ctx context.Context, req *rpc.CreateApiReques
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
-	_ = s.notify(ctx, rpc.Notification_CREATED, name.String())
+	s.notify(ctx, rpc.Notification_CREATED, name.String())
```

Comment:
Is this linter-related churn? I thought I saw some assignments to "_" show up when the linter CI was added.

Reply:
The linter enforces that we explicitly discard errors if they're returned, so you're right that I added those recently. I updated the method to not return an error at all, which is why you're seeing the explicit discard removed from every call. It's not really related to logging so maybe it belongs in a different PR, but it was a trivial and non-functional change.

My assumption is that we don't want users to receive something like an internal error when something goes wrong publishing a notification, so we can just log the errors without returning them. I added those logs in this PR.

Reply:
Agreed - thanks for the explanation!